missing-values-imputer/wind-imputation/utils.py

```diff
@@ -0,0 +1,9 @@
+from ydata.dataset import Dataset
+
+
+def preprocess_data(data: Dataset):
```

Comment:
Shared name with the other preprocess method. Should this one be named more in the line of  _setting_ and _sorting_ index?

Reply:
Yep, I'm going to rename it